lib/blockscore/connection.rb

```diff
@@ -1,5 +1,11 @@
 module BlockScore
   module Connection
+    MAJOR_VERSION = BlockScore::VERSION.to_i
+    REPO          = 'https://github.com/BlockScore/blockscore-ruby'
+    ACCEPT_HEADER = "application/vnd.blockscore+json;version=#{MAJOR_VERSION}"
+    USER_AGENT    = "blockscore-ruby/#{BlockScore::VERSION} (#{REPO})"
+    CONTENT_TYPE  = 'application/json'
```

Comment:
@backus I looked in your style guide. Normally I also freeze strings - especially on an API that won't soon be based on Ruby 3.x ONLY). But there was nothing in the style guide, pro or con, so I am going to follow @dkubb guidance, because that is what I also do.